plugins/net_plugin/net_plugin.cpp

```diff
@@ -1239,6 +1242,19 @@ namespace eosio {
       return create_send_buffer( packed_transaction_which, trx );
    }
 
+   static std::shared_ptr<std::vector<char>> create_send_buffer( const signed_block_v0_ptr& sb ) {
+      // this implementation is to avoid copy of signed_block to net_message
+      // matches which of net_message for signed_block
```

Comment:
Fix comment to say "signed_block_v0" in both lines.

Reply:
Fixed.